### DISCUSSION TITLE
Show apps if nothing is typed

### DISCUSSION THREAD

**Discussion by dougg0k** [Ulauncher/Ulauncher#1415] [Q&A] [answered]
Hi,

Is there a way to show all or some apps if nothing is typed? Like you see in a launcher.

In some way like https://user-images.githubusercontent.com/101165014/157171778-9f86a297-317c-4424-a5c5-d12b0866a677.png

**Comment by friday** [comment, marked as answer]
You can make it show your most launched apps:

![image](https://github.com/user-attachments/assets/ce4bbd39-926f-4b4e-ac7e-dcc0d9e53ffa)
